packages/typescript-compat/typescript-compat.mjs

```diff
@@ -1,48 +0,0 @@
-#!/usr/bin/env node
-
-// Copyright 2021-2024 Buf Technologies, Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-import { readdirSync } from "node:fs";
-import { join as joinPath, basename } from "node:path";
-import { execSync } from "node:child_process";
-import { stdout, stderr, exit } from "node:process";
-
-const typescriptCompatDir = new URL(".", import.meta.url).pathname;
```

Comment:
This script is replaced with the command `turbo run test -F './packages/typescript-compat/*'`.